assignments/assignment-6/public/scripts.js

```diff
@@ -204,6 +228,49 @@ $(document).ready(function() {
       loadAdminQuestionForm(pointValue);
     }
   });
+  
+  $(".question-point-block").click(function() {
+  if ($(".question-window").css("display") == "none") {
```

Comment:
So here we are checking: if this class has display: none, do some stuff. Won't this mean we can keep clicking and answering same question over and over again?

Reply:
What this does is that it makes sure you can click questions only if the question window is closed.

Reply:
It's possible to add a check to stop the user from clicking the same question, I just skipped it since I saw no requirement for it. I can add it in if you want.

Reply:
Yeah, users should not be able to click and answer the same question again.